src/components/DropDown.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+import Select from 'react-select';
```

Comment:
This is an interesting choice. Can you speak a little bit as to why you went this route, instead of using https://material-ui.com/components/selects/#selects? You might have experienced something that I missed completely and I'd love to know more :D

Reply:
https://material-ui.com/components/autocomplete/#react-select
I found this from the material ui recommendation. It's pretty well support library.

Reply:
Good answer.